pages/index.tsx

```diff
@@ -44,9 +53,33 @@ export default function Home() {
         </div>
         <div className="grid call-container mobile-socmed-call socmed-call w-fit mx-auto md:mx-5 mt-44 sm:mt-36 md:mt-24 lg:mt-28 xl:mt-28 ">
           <div className="flex icons">
-            <SocialIcons imgURL="/images/socials/instagram.png" alt="Instagram" />
-            <SocialIcons imgURL="/images/socials/facebook.png" alt="Facebook" />
-            <SocialIcons imgURL="/images/socials/youtube.png" alt="Youtube" />
+            <a
+              href="https://www.instagram.com/tmifellowship"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <SocialIcons
+                imgURL="/images/socials/instagram.png"
+                alt="Instagram"
+              />
+            </a>
+            <a
+              href="https://www.facebook.com/tmifellowship"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <SocialIcons
+                imgURL="/images/socials/facebook.png"
+                alt="Facebook"
+              />
+            </a>
+            <a
+              href="https://www.youtube.com/channel/UCXMRU2-nQZwMNQyjM64mLJA"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <SocialIcons imgURL="/images/socials/youtube.png" alt="Youtube" />
+            </a>
```

Comment:
use Link nextjs component if possible

Reply:
diba diay ang Link nga component kay gamiton ra for web pages nga ga exists sa web app?

Reply:
oh okok ana diay na, mbmb